parm/atm/obs/config/sfcship.yaml

```diff
@@ -0,0 +1,77 @@
+obs space:
+  name: sfcship
+  obsdatain:
+    engine:
+      type: H5File
+      obsfile: $(OBS_DIR)/$(OBS_PREFIX)sfcship.$(OBS_DATE).nc4
+  obsdataout:
+    engine:
+      type: H5File
+      obsfile: $(DIAG_DIR)/diag_sfcship_$(OBS_DATE).nc4
+      overwrite: true
+  io pool:
+    max pool size: 1
+  simulated variables: [surface_pressure, air_temperature, specific_humidity]
+get values:
+  interpolation method: $(INTERP_METHOD)
+obs operator:
+  name: Composite
+  components:
+    - name: VertInterp
+      variables:
+      - name: air_temperature
+      - name: specific_humidity
+    - name: SfcPCorrected
+      variables:
+      - name: surface_pressure
+      da_psfc_scheme: GSI
+      geovar_sfc_geomz: surface_geometric_height
+      geovar_geomz: geopotential_height
+linear obs operator:
+  name: Identity
+
+obs filters:
+  # Observation range sanity check
+  - filter: Bounds Check
+    filter variables:
+    - name: surface_pressure
+    minvalue: 37499
```

Comment:
Based on our Wednesday group discussion, I think we need to make clear these are 'floats' and not integers (throughout this YAML).